packages/slate-editor/src/extensions/floating-add-menu/components/Dropdown.tsx

```diff
@@ -196,7 +196,7 @@ export function Dropdown<Action>({
                                                 option.isBeta ? styles.warning : styles.primary,
```

Comment:
Should we also tweak the style of the badge? Like make it [gray-ish instead of yellow](https://prezly.slack.com/archives/C08PUBNGY/p1687523209208189?thread_ts=1687429136.303879&cid=C08PUBNGY)?

Reply:
e974a046ce96f5c254921156ae573d407b64d4a3